src/displayapp/screens/Notifications.cpp

```diff
@@ -176,7 +176,17 @@ Notifications::NotificationItem::NotificationItem(const char* title,
   lv_obj_set_style_local_text_color(alert_type, LV_LABEL_PART_MAIN, LV_STATE_DEFAULT, lv_color_hex(0x888888));
   if (title == nullptr)
     title = "Notification";
+  char* pchar;
+  pchar = strchr(title, '\n');
+  while (pchar != nullptr) {
+    *pchar = ' ';
+    pchar =
```

Comment:
Can you explain this row?

Reply:
It searches for new lines and replaces them with spaces. 
`strchr()` gives a pointer to the first occurrence of the newline or nullptr if none is found. the while loop keeps searching for newlines until none are left.

Reply:
![img](https://cdn.discordapp.com/attachments/622348681538043924/842300570713980948/IMG_20210513_092055.jpg)

it prevents this from happening

Reply:
I meant more the assignment that seems weird and throws a warning.